R/pkg/R/group.R

```diff
@@ -243,17 +236,73 @@ setMethod("gapply",
           signature(x = "GroupedData"),
           function(x, func, schema) {
             try(if (is.null(schema)) stop("schema cannot be NULL"))
```

Comment:
I think we use `stop` or `warn` directly in SparkR without wrapping in a `try` block. I think the `try` block is useful if we want to catch an error of one kind and then reformat it or show a different error etc. In this case `stop` should be sufficient.